.obs/dockerfile/slem4r-os/Dockerfile

```diff
@@ -0,0 +1,97 @@
+# SPDX-License-Identifier: Apache-2.0
+# Define the names/tags of the container
+#!BuildTag: suse/sle-micro-rancher/%%SLEMICRO_VERSION%%:latest
+#!BuildTag: suse/sle-micro-rancher/%%SLEMICRO_VERSION%%:%RELEASE%
+#!BuildConstraint: hardware:disk:size unit=G 8
+#
+
+FROM suse/sle15:15.5 as host
+
+MAINTAINER SUSE LLC (https://www.suse.com/)
+
+# Define labels according to https://en.opensuse.org/Building_derived_containers
+# labelprefix=com.suse.sle.micro.rancher
+LABEL org.opencontainers.image.title="SLE Micro for Rancher"
+LABEL org.opencontainers.image.description="Image containing SLE Micro for Rancher - a containerized OS layer for Kubernetes."
+LABEL org.opencontainers.image.version="%%SLEMICRO_VERSION%%.%RELEASE%"
+LABEL org.opencontainers.image.url="https://www.suse.com/products/micro/"
+LABEL org.opencontainers.image.created="%BUILDTIME%"
+LABEL org.opencontainers.image.vendor="SUSE LLC"
+LABEL org.opencontainers.image.source="%SOURCEURL%"
+LABEL org.opensuse.reference="registry.suse.com/suse/sle-micro-rancher/%%SLEMICRO_VERSION%%:%RELEASE%"
+LABEL org.openbuildservice.disturl="%DISTURL%"
+LABEL com.suse.supportlevel="alpha"
+LABEL com.suse.eula="sle-eula"
+LABEL com.suse.lifecycle-url="https://www.suse.com/lifecycle#suse-linux-enterprise-micro"
+LABEL com.suse.image-type="sle-micro"
+LABEL com.suse.release-stage="unreleased"
+# endlabelprefix
+
+RUN mkdir /osimage
+
+RUN rpm --initdb --root /osimage
+
+RUN zypper --installroot /osimage in --no-recommends -y filesystem
+
+# make system bootable
+RUN zypper --installroot /osimage in --no-recommends -y grub2 shim dracut kernel kernel-firmware-all systemd bash
+
+#!ArchExclusiveLine: x86_64
+RUN if [ `uname -m` = "x86-64" ]; then zypper --installroot /osimage in --no-recommends -y syslinux; fi
+
+# make dracut happy
+RUN zypper --installroot /osimage in --no-recommends -y squashfs NetworkManager device-mapper iproute2 tar curl ca-certificates ca-certificates-mozilla
+
+# make ARM happy
+#!ArchExclusiveLine: aarch64
+RUN if [ `uname -m` = "aarch64" ]; then zypper --installroot /osimage in -y raspberrypi-firmware raspberrypi-firmware-config raspberrypi-firmware-dt u-boot-rpiarm64 grub2-arm64-efi; fi
+                
+# make SUSE happy
+RUN zypper --installroot /osimage in --no-recommends -y SLE-Micro-Rancher-release systemd-presets-branding-SLE-Micro-for-Rancher
+
+# make elemental-register happy
+RUN zypper --installroot /osimage in --no-recommends -y dmidecode lvm2
+
+# make Rancher (containerd) happy
+RUN zypper --installroot /osimage in --no-recommends -y apparmor-parser
+
+# add elemental
+RUN zypper --installroot /osimage in --no-recommends -y elemental
+
+# make users happy
+RUN zypper --installroot /osimage in --no-recommends -y procps openssl openssh vim-small less iputils kernel-firmware-all NetworkManager-wwan cryptsetup podman zypper
```

Comment:
Is zypper going to work anyway given the read-only filesystem?

Reply:
There's no read-only filesystm ad this point. It's a sub-directory (`/osimage`) in a Docker build 😉

Reply:
No I mean why we do install zypper as dependency.
And shouldn't zypper already be included anyway?

Reply:
Oh I see it now, so we don't copy the entire `host` container, but you do a selection, this explains why we install the kernel and some other dependencies. So indeed zypper is not going to be included, and you need to run the clean later. 
Ok.